package.json

```diff
@@ -1,24 +1,40 @@
 {
   "name": "rapid-experiments-shared",
+  "description": "This is a place to define data and schemas used across the rapid experiments program.",
   "version": "0.0.1",
-  "license": "MPL-2.0",
-  "scripts": {
-    "build": "npm-run-all -s build:*",
-    "build:json-schema": "./bin/build-schemas.ts",
-    "artifact": "./bin/pack-artifact.sh",
-    "test": "npm-run-all -s test:*",
-    "test:normandy-arguments": "./test/normandy-arguments.ts"
+  "author": "Mozilla",
+  "bugs": {
+    "url": "https://github.com/mozilla/rapid-experiments-shared/issues"
   },
   "dependencies": {},
   "devDependencies": {
     "@types/node": "^14.0.3",
     "@types/node-fetch": "^2.5.7",
+    "@typescript-eslint/eslint-plugin": "^3.0.2",
+    "@typescript-eslint/parser": "^3.0.2",
     "ajv": "^6.12.2",
+    "eslint": "^7.1.0",
     "node-fetch": "^2.6.0",
     "npm-run-all": "^4.1.5",
     "ts-json-schema-generator": "^0.68.0",
     "ts-node": "^8.10.1",
     "typescript": "^3.9.3",
     "typescript-json-schema": "^0.42.0"
+  },
+  "homepage": "https://github.com/mozilla/rapid-experiments-shared#readme",
+  "keywords": [],
+  "license": "MPL-2.0",
+  "private": true,
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/mozilla/rapid-experiments-shared.git"
+  },
```

Comment:
Sorry, a lot of this was the fallout of a `npm init -y` which adds this boilerplate. 
Also added in that `private: true` to avoid accidental npm publish.

Reply:
Can you revert these changes? It makes this PR a lot more complicated to review than it needs to be.

The private key is a good call though. Although we will eventually publish to NPM, we're not ready for that just yet.